sixMans/game.py

```diff
@@ -585,7 +585,7 @@ async def post_more_lobby_info(self, helper_role=None, invalid=False):
                 "the `winning_team` parameter is either `Blue` or `Orange`. Both teams will need to verify the results.\n\nIf you wish to cancel "
                 "the game and allow players to queue again you can use the `{0}cg` command. Both teams will need to verify that they wish to "
                 "cancel the game.".format(self.prefix), inline=False)
-        help_message = "If you think the bot isn't working correctly or have suggestions to improve it, please contact adammast."
+        help_message = "If you think the bot isn't working correctly or have suggestions to improve it, please contact the RSC Development Committee."
```

Comment:
This probably is an improvement over adammast considering we'll be maintaining it now. I wonder if we can put out a more specific point of contact, or info on how to contact us. Probably good for now.

Reply:
Yeah, at some point we'll probably want to expose some kind of ticketing system or something to the users.